src/main/java/se/vandmo/dependencylock/maven/mojos/CheckMojo.java

```diff
@@ -26,20 +27,36 @@ public void execute() throws MojoExecutionException {
       return;
     }
 
-    DependenciesLockFileAccessor lockFile = lockFile();
+    LockFileAccessor lockFile = lockFile();
     if (!lockFile.exists()) {
       throw new MojoExecutionException(
           "No lock file found, create one by running 'mvn"
               + " se.vandmo:dependency-lock-maven-plugin:lock'");
     }
-    LockedDependencies lockedDependencies =
-        format().dependenciesLockFile_from(lockFile, pomMinimums(), getLog()).read();
-    LockedDependencies.Diff diff = lockedDependencies.compareWith(projectDependencies(), filters());
-    if (diff.equals()) {
-      getLog().info("Actual dependencies matches locked dependencies");
+    if (alsoLockBuild()) {
```

Comment:
I think it should default to checking the build if it exists in the lock file. An opt-**out** flag might be needed in the future but better to skip that until somebody actually needs it.

Reply:
Fair point. Done in https://github.com/vandmo/dependency-lock-maven-plugin/pull/85/commits/9a417a4ceed44f743e5faecf8b522a519b4e1c0a